src/js/components/stock-movement-wizard/StockMovementVerifyRequest.jsx

```diff
@@ -0,0 +1,338 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+import { getTranslate } from 'react-localize-redux';
+
+import CreateStockMovement from './outbound/CreateStockMovement';
+import AddItemsPage from './outbound/AddItemsPage';
+import EditPage from './request/EditPage';
+import PickPage from './outbound/PickPage';
+import PackingPage from './outbound/PackingPage';
+import SendMovementPage from './outbound/SendMovementPage';
+import Wizard from '../wizard/Wizard';
+import apiClient from '../../utils/apiClient';
+import { showSpinner, hideSpinner, fetchTranslations, updateBreadcrumbs, fetchBreadcrumbsConfig } from '../../actions';
+import { translateWithDefaultMessage } from '../../utils/Translate';
+
+import './StockMovement.scss';
+
+/** Main Verify Request Stock movement form's wizard component. */
+class StockMovementsVerifyRequest extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      values: this.props.initialValues,
+      currentPage: 1,
+    };
+
+    this.updateWizardValues = this.updateWizardValues.bind(this);
+  }
+
+  componentDidMount() {
+    this.props.fetchBreadcrumbsConfig();
+    this.props.fetchTranslations('', 'stockMovement');
+
+    if (this.props.stockMovementTranslationsFetched) {
+      this.dataFetched = true;
+
+      this.fetchInitialValues();
+    }
+
+    const {
+      actionLabel, defaultActionLabel, actionUrl, listLabel, defaultListLabel, listUrl,
+    } = this.props.breadcrumbsConfig;
+    this.props.updateBreadcrumbs([
+      { label: listLabel, defaultLabel: defaultListLabel, url: listUrl },
+      { label: actionLabel, defaultLabel: defaultActionLabel, url: actionUrl },
+    ]);
+  }
+
+  componentWillReceiveProps(nextProps) {
+    if (this.props.locale && this.props.locale !== nextProps.locale) {
+      this.props.fetchTranslations(nextProps.locale, 'stockMovement');
+    }
+
+    if (nextProps.stockMovementTranslationsFetched && !this.dataFetched) {
+      this.dataFetched = true;
+
+      this.fetchInitialValues();
+    }
+
+    if (nextProps.breadcrumbsConfig &&
+      nextProps.breadcrumbsConfig !== this.props.breadcrumbsConfig) {
+      const {
+        actionLabel, defaultActionLabel, actionUrl, listLabel, defaultListLabel, listUrl,
+      } = nextProps.breadcrumbsConfig;
+
+      this.props.updateBreadcrumbs([
+        { label: listLabel, defaultLabel: defaultListLabel, url: listUrl },
+        { label: actionLabel, defaultLabel: defaultActionLabel, url: actionUrl },
+      ]);
+    }
+  }
+
+  /**
+   * Returns array of form steps.
+   * @public
+   */
+  getStepList() {
+    let stepList = [];
+    if (this.props.hasPackingSupport) {
+      stepList = [this.props.translate('react.stockMovement.create.label', 'Create'),
```

Comment:
I think we don't need create and add items pages steps in this workflow.

Reply:
You're right. I'll remove redundant steps.

Reply:
done